lib/certification/reso-common-format/README

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
@mohit-s96 - we should add a README too similar to the DD 2.0 README. We need a new section on the root README about how to test RCF for certification, not just schema validation. 

It should link off to its own page that explains how to use the tools and describes the files and outcomes. https://github.com/RESOStandards/reso-certification-utils/blob/main/lib/certification/data-dictionary/README.md


For DD 2.0 variations and schema validation testing, you could potentially just link to the sections from the current DD 2.0 testing guide.